spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1395,7 +1412,6 @@ def name_id_version(format_urn)
             <samlp:NameIDPolicy AllowCreate="true" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"/>
             <samlp:RequestedAuthnContext Comparison="exact">
           <saml:AuthnContextClassRef>urn:gov:gsa:ac:classes:sp:PasswordProtectedTransport:duo</saml:AuthnContextClassRef>
-              <saml:AuthnContextClassRef>http://idmanagement.gov/ns/assurance/ial/1</saml:AuthnContextClassRef>
```

Comment:
What's the relevance of this change?

Reply:
it just makes the analytics hash that is being matched a little simpler. i can add it back if you prefer, but since the saml code flow has a default ial it shouldn't be necessary in the assertion that's being passed through.

Reply:
No strong preference, just wasn't sure its relevance, makes sense 👍